radar_compose.yaml

```diff
@@ -0,0 +1,36 @@
+services:
+
+  ros1_master:
+    image: "ghcr.io/ipab-rad/av_radar:1.0.0"
+    container_name: ros1_master
+    network_mode: "host"
+    privileged: true
+    command: rosmaster --core
+    volumes:
+      - "/dev/shm:/dev/shm"
```

Comment:
Minor nitpick: This container won't need sharing this volume. But happy if all containers share this volume by default (Even if they run ROS1)

Reply:
True, this volume is not needed, I rather have mounting only when necessary, will remove it.

Reply:
Done